src/lib/__snapshots__/create-config.test.ts.snap

```diff
@@ -95,7 +94,7 @@ exports[`should create default config 1`] = `
       "proPlanAutoCharge": false,
       "responseTimeWithAppNameKillSwitch": false,
       "segmentContextFieldUsage": false,
-      "strategySplittedButton": false,
+      "strategySplittedButton": true,
```

Comment:
What's going on here? Doesn't look related?

Reply:
Thank you, shouldn't have made it into this PR